.github/workflows/auto-build-rpm.yml

```diff
@@ -29,6 +29,19 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v2
 
+      - uses: docker/setup-buildx-action@v1
+        with:
+          install: true
+
+      - run: docker login
```

Comment:
It seems that we don't provide the username and password when docker login?

Reply:
there exist credentials in the actions environment, see this step's result in https://github.com/apache/apisix-dashboard/pull/2072/checks?check_run_id=3354174287.
I add this step to solve requests denied to access docker images, but it doesn't work. Maybe this PR has no authority to access private docker images?

Reply:
@qsliu2017  we don't need to login here.

Reply:
@nic-chen Ok, I've removed it. But why docker request is denied?